Using Cartridge 2.5.0+ requires additional setup

Suppose you have in the rockspec of your Cartridge application version greater than 2.5.0-1. In that case when installing dependencies, `cartridge build` requires additional dependencies which are not installed:

(When installing luagraphqlparser as a dependency of cartridge):
```
Warning: unmatched variable TARANTOOL_DIR

-- The C compiler identification is GNU 4.8.5

-- The CXX compiler identification is unknown

-- Check for working C compiler: /usr/bin/cc

-- Check for working C compiler: /usr/bin/cc -- works

-- Detecting C compiler ABI info

-- Detecting C compiler ABI info - done

CMake Error: your CXX compiler: "CMAKE_CXX_COMPILER-NOTFOUND" was not found.   Please set CMAKE_CXX_COMPILER to a valid compiler path or name.

-- Found TARANTOOL: /usr/include (found version "2.4.3-1-g986fab7") 

-- Tarantool LUADIR is /app/.rocks/share/tarantool/rocks/luagraphqlparser/0.1.0-1/lua

-- Tarantool LIBDIR is /app/.rocks/share/tarantool/rocks/luagraphqlparser/0.1.0-1/lib

-- Configuring incomplete, errors occurred!

See also "/tmp/luarocks_luagraphqlparser-0.1.0-1-6XEBQ6/luagraphqlparser/build.luarocks/CMakeFiles/CMakeOutput.log".

See also "/tmp/luarocks_luagraphqlparser-0.1.0-1-6XEBQ6/luagraphqlparser/build.luarocks/CMakeFiles/CMakeError.log".



Error: Failed installing dependency: http://rocks.tarantool.org/cartridge-2.5.0-1.all.rock - Failed installing dependency: http://rocks.tarantool.org/graphql-0.1.0-1.src.rock - Failed installing dependency: http://rocks.tarantool.org/luagraphqlparser-0.1.0-1.rockspec - Build error: Failed cmake.

   ⨯ Failed to install rocks: Failed to run 

/usr/bin/tarantoolctl rocks make
```

--- a/src/main/resources/Dockerfile
+++ b/src/main/resources/Dockerfile
@@ -11,7 +11,7 @@ ENV TARANTOOL_WORKDIR=$TARANTOOL_WORKDIR
 ENV TARANTOOL_RUNDIR=$TARANTOOL_RUNDIR
 ENV TARANTOOL_DATADIR=$TARANTOOL_DATADIR
 RUN curl -L https://tarantool.io/installer.sh | VER=$TARANTOOL_VERSION /bin/bash -s -- --repo-only && \
-    yum -y install cmake make gcc gсс-с++ git unzip tarantool tarantool-devel cartridge-cli && \
+    yum -y install cmake make gcc gcc-c++ git unzip tarantool tarantool-devel cartridge-cli && \
     yum clean all
 RUN groupadd -g $TARANTOOL_SERVER_GID $TARANTOOL_SERVER_GROUP && \
     useradd -u $TARANTOOL_SERVER_UID -g $TARANTOOL_SERVER_GID -m -s /bin/bash $TARANTOOL_SERVER_USER \

--- a/src/test/resources/cartridge/app/roles/api_router.lua
+++ b/src/test/resources/cartridge/app/roles/api_router.lua
@@ -1,8 +1,4 @@
-local crud = require('crud')
 local cartridge = require('cartridge')
-
--- initialize crud
-crud.init()
 
 local function profile_return(a, b, c, d, e, f)
     return a, b, c, d, e, f;
@@ -93,67 +89,9 @@ local function get_schema_metadata()
 	return router:callro(bucket_id, 'storage_get_space_format', {})
 end
 
--- crud cluster API call
-
-local function crud_cluster_api_get(space_name, key, opts)
-	return crud.get(space_name, key, opts)
-end
-
-local function crud_cluster_api_insert(space_name, key_parts, obj, opts)
-	return crud.insert(space_name, key_parts, obj, opts)
-end
-
-local function crud_cluster_api_delete(space_name, key, opts)
-	return crud.delete(space_name, key, opts)
-end
-
-local function crud_cluster_api_replace(space_name, key_parts, obj, opts)
-	return crud.replace(space_name, key_parts, obj, opts)
-end
-
-local function crud_cluster_api_update(space_name, key, operations, opts)
-	return crud.update(space_name, key, operations, opts)
-end
-
-local function crud_cluster_api_upsert(space_name, key_parts, obj, operations, opts)
-	return crud.upsert(space_name, key_parts, obj, operations, opts)
-end
-
-local function crud_cluster_api_select(space_name, key_parts, opts)
-	local objects = {}
-
-    local iter, err = crud.select(space_name, key_parts, opts)
-    if err ~= nil then return nil, err end
-
-	while iter:has_next() do
-		local obj = iter:get()
-		assert(obj ~= nil)
-
-		table.insert(objects, obj)
-	end
-
-	return unpack(objects)
-end
-
-local function crud_cluster_api_get_schema()
-	return crud.get_schema()
-end
-
 local function init(opts)
     if opts.is_master then
     end
-
-
-	rawset(_G, 'crud_cluster_api_get_schema', crud_cluster_api_get_schema)
-
-	rawset(_G, 'crud_cluster_api_get', crud_cluster_api_get)
-	rawset(_G, 'crud_cluster_api_insert', crud_cluster_api_insert)
-	rawset(_G, 'crud_cluster_api_delete', crud_cluster_api_delete)
-	rawset(_G, 'crud_cluster_api_replace', crud_cluster_api_replace)
-	rawset(_G, 'crud_cluster_api_update', crud_cluster_api_update)
-	rawset(_G, 'crud_cluster_api_upsert', crud_cluster_api_upsert)
-	rawset(_G, 'crud_cluster_api_select', crud_cluster_api_select)
-
 
 	rawset(_G, 'profile_return', profile_return)
 
@@ -163,9 +101,6 @@ local function init(opts)
     rawset(_G, 'profile_update', profile_update)
     rawset(_G, 'profile_upsert', profile_upsert)
     rawset(_G, 'profile_get', profile_get)
-	
-
-    rawset(_G, 'get_schema_metadata', get_schema_metadata)
 
     return true
 end

--- a/src/test/resources/cartridge/testapp-scm-1.rockspec
+++ b/src/test/resources/cartridge/testapp-scm-1.rockspec
@@ -7,10 +7,10 @@ source  = {
 dependencies = {
     'tarantool',
     'lua >= 5.1',
-    'checks == 3.0.1-1',
-    'cartridge == 2.1.2-1',
-    'ddl == 1.1.0-1',
-    'crud == 0.1.0-1',
+    'checks == 3.1.0-1',
+    'cartridge == 2.6.0-1',
+    'ddl == 1.4.0-1',
+    'crud == 0.8.0-1',
 }
 build = {
     type = 'none';

